lib/features/sheet_botton/coffee_sheet_bottom.dart

```diff
@@ -8,23 +8,24 @@ class CoffeeSheetBottom extends StatelessWidget {
   final Widget body;
   final Color? backgroundColor;
   final CoffeeButton button;
-  final double? barrierOpacity;
+  final bool hasBarrierOpacity;
   const CoffeeSheetBottom({
     super.key,
     required this.title,
     required this.body,
     this.alignmentTitle,
     this.backgroundColor,
     required this.button,
-    this.barrierOpacity,
+    this.hasBarrierOpacity = true,
   });
 
   void show(BuildContext context) {
     showModalBottomSheet<void>(
       context: context,
       isScrollControlled: true,
-      barrierColor:
-          ThemeService.of.backgroundColor.withOpacity(barrierOpacity ?? 0.3),
+      barrierColor: hasBarrierOpacity
+          ? Theme.of(context).bottomSheetTheme.modalBarrierColor
```

Comment:
ThemeService.of.backgroundColor.withOpacity(0.3) adicionar essa linha no lugar da modalBarrierColor